src/main/java/life/qbic/data/processing/registration/RegistrationConfiguration.java

```diff
@@ -18,7 +18,7 @@ public RegistrationConfiguration(String workingDirectory, String targetDirectory
     if (!workingDirectory().toFile().isDirectory()) {
       throw new IllegalArgumentException(targetDirectory + " is not a directory");
     }
-    this.targetDirectory = Paths.get(Objects.requireNonNull(targetDirectory, "targetDirectory must not be null"));
+    this.targetDirectory = Paths.get(Objects.requireNonNull(targetDirectory, "targetDirectories must not be null"));
```

Comment:
Should the requirement(s) here not be on empty instead of null? I guess the path could be the current folder, but it is unlikely and coincides with an empty property, no?

Reply:
I think the application should not make assumptions about what a possible user's configuration wish might be. We leave the "meaningful" configuration settings to the user. We only check if it is not null (fail fast) and if the file exists.